pom.xml

```diff
@@ -176,6 +176,12 @@
         <jodah-failsafe.version>0.9.5</jodah-failsafe.version>
         <apache.shiro.version>1.3.2</apache.shiro.version>
         <spark.version>2.5.5</spark.version>
+
+        <!-- gitflow-incremental-builder -->
+        <!-- See https://github.com/vackosar/gitflow-incremental-builder for a list of all the options -->
+        <gib.referenceBranch>refs/remotes/origin/master</gib.referenceBranch>
```

Comment:
I do know several engineers who do things oddly, making their forks origin. I'm not sure if they at least stick with the name `origin`. This might require some documentation and a little dissemination of usage.